app/src/test/java/io/apicurio/registry/rest/v2/GroupsResourceTest.java

```diff
@@ -314,38 +315,44 @@ public void testCreateArtifact() throws Exception {
                 .body("id", equalTo("testCreateArtifact/EmptyAPI-customName"))
                 .body("type", equalTo(ArtifactType.OPENAPI.name()));
 
-        // Create OpenAPI artifact - provide a custom No-ASCII name
-        String customNoASCIIName = new String(Base64.decode("Q1VTVE9NIE5BTUUgd2l0aCBOTy1BU0NJSSBjaGFyIMSN"), StandardCharsets.UTF_8);
+        // Create OpenAPI artifact - provide a custom description
+        String customDescription = "CUSTOM DESCRIPTION";
         given()
             .when()
                 .contentType(CT_JSON + "; artifactType=OPENAPI")
                 .pathParam("groupId", GROUP)
-                .header("X-Registry-ArtifactId", "testCreateArtifact/EmptyAPI-customNameEncoded")
-                .header("X-Registry-Name-Encoded", Base64.encode(customNoASCIIName.getBytes(StandardCharsets.UTF_8)))
+                .header("X-Registry-ArtifactId", "testCreateArtifact/EmptyAPI-customDescription")
+                .header("X-Registry-Description", customDescription)
                 .body(artifactContent)
                 .post("/registry/v2/groups/{groupId}/artifacts")
             .then()
                 .statusCode(200)
                 .body("groupId", equalTo(GROUP))
-                .body("name", equalTo(customNoASCIIName))
-                .body("id", equalTo("testCreateArtifact/EmptyAPI-customNameEncoded"))
+                .body("description", equalTo(customDescription))
+                .body("id", equalTo("testCreateArtifact/EmptyAPI-customDescription"))
                 .body("type", equalTo(ArtifactType.OPENAPI.name()));
+    }
 
-        // Create OpenAPI artifact - provide a custom description
-        String customDescription = "CUSTOM DESCRIPTION";
+    @Test
+    @DisabledIfEnvironmentVariable(named = CURRENT_ENV, matches = CURRENT_ENV_MAS_REGEX)
+    public void testCreateArtifactNoAscii() {
+        String artifactContent = resourceToString("openapi-empty.json");
+
+        // Create OpenAPI artifact - provide a custom No-ASCII name
+        String customNoASCIIName = "CUSTOM NAME with NO-ASCII char č";
```

Comment:
I still think it might be better to hard-code a base64 encoded value here because having a unicode character in the source file may be problematic in git and in some IDEs.  For example I think Eclipse on Windows used to default to some non-UTF charset.  I don't feel very strongly about it though.